components/ai_chat/resources/custom_site_distiller_scripts/scripts/github.com/distiller.ts

```diff
@@ -7,13 +7,13 @@ import { LEO_DISTILLATION_LEVEL } from '../distillation'
 import { distillBranches } from './branches'
 import { GetPageType, SupportedPage } from './utils'
 
-let _DISTILLATION_LEVEL = LEO_DISTILLATION_LEVEL.LOW
+let _DISTILLATION_LEVEL = LEO_DISTILLATION_LEVEL.FULL
 
 export function getDistillationLevel() {
   return _DISTILLATION_LEVEL
 }
 
-export default function distill(level: LEO_DISTILLATION_LEVEL) {
+export default function distill(level = LEO_DISTILLATION_LEVEL.FULL) {
```

Comment:
This was mistakenly left out of the original PR; there should be a default distillation level.